### PR TITLE
SDI-386 Processed a visit change event

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt [hunk 1]
```diff
@@ -4,7 +4,11 @@ import com.amazonaws.services.sqs.AmazonSQS
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
-fun prisonVisitCreatedMessage(visitId: String = "12", prisonerId: String = "AB12345", occurredAt: String = "2021-03-05T11:23:56.031Z") = """
+fun prisonVisitCreatedMessage(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt:7]: just formatting

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt [hunk 2]
```diff
@@ -18,7 +22,11 @@ fun prisonVisitCreatedMessage(visitId: String = "12", prisonerId: String = "AB12
         "SigningCertURL": "https://sns.us-east-1.amazonaws.com/SimpleNotificationService-0000000000000000000000.pem"}      
 """.trimIndent()
 
-fun prisonVisitCancelledMessage(visitId: String = "12", prisonerId: String = "AB12345", occurredAt: String = "2021-03-05T11:23:56.031Z") = """
+fun prisonVisitCancelledMessage(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt:25]: just formatting

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/integration/VisitToNomisTest.kt [hunk 3]
```diff
@@ -38,24 +43,24 @@ class VisitToNomisTest : SqsIntegrationTestBase() {
     await untilCallTo { nomisApi.postCountFor("/prisoners/A32323Y/visits") } matches { it == 1 }
 
     nomisApi.verify(
-      WireMock.postRequestedFor(WireMock.urlEqualTo("/prisoners/A32323Y/visits"))
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/integration/VisitToNomisTest.kt:41]: just import

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt [hunk 4]
```diff
@@ -32,6 +40,24 @@ fun prisonVisitCancelledMessage(visitId: String = "12", prisonerId: String = "AB
...
+  visitId: String = "12",
+  prisonerId: String = "AB12345",
+  occurredAt: String = "2021-03-05T11:23:56.031Z"
+) = """
+      {
+        "Type": "Notification", 
+        "MessageId": "48e8a79a-0f43-4338-bbd4-b0d745f1f8ec", 
+        "Token": null, 
+        "TopicArn": "arn:aws:sns:eu-west-2:000000000000:hmpps-domain-events", 
+        "Message": "{\"eventType\":\"prison-visit.changed\", \"prisonerId\": \"$prisonerId\", \"occurredAt\": \"$occurredAt\", \"additionalInformation\": {\"reference\": \"$visitId\"}}",
```
Comment (lbennett-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/helpers/Messages.kt:53]: didn't they go for revised?

**Review by lbennett-moj — APPROVED** [review]

**Pull request merged**